diff_drive_controller/include/diff_drive_controller/diff_drive_controller.h

```diff
@@ -123,6 +123,23 @@ namespace diff_drive_controller
       wheel_speed_limiter_ = wheel_speed_limiter;
     }
 
+  protected:
+    /*
+     * \brief Default wheel speed limiter function
+     * \param [in, out] left_velocity  Left wheel velocity (not used)
+     * \param [in, out] right_velocity Right wheel velocity (not used)
+     */
+    virtual void wheelSpeedLimiter(double& left, double& right)
+    {
+      if (wheel_speed_limiter_)
```

Comment:
This is good, but I personally thing we can just remove the old way to set the wheel speed limits. I don't think it'd break anything we cannot upgrade to the new API simultaneously. It'd be much cleaner:

1. This would be empty
2. We'd remove the `setWheelSpeedLimiter` setter
3. We'd remove the `wheel_speed_limiter_` attribute, and anything that was added/included to define it.

I guess the change to support inheritance is not trivial, or maybe it is? But I'd like to have this done for the sake of having a cleaner API for this thing.